pkg/skaffold/parser/config.go

```diff
@@ -80,7 +81,7 @@ func GetAllConfigs(ctx context.Context, opts config.SkaffoldOptions) ([]schemaUt
 func GetConfigSet(ctx context.Context, opts config.SkaffoldOptions) (SkaffoldConfigSet, error) {
 	cOpts := configOpts{file: opts.ConfigurationFile, selection: nil, profiles: opts.Profiles, isRequired: false, isDependency: false}
 	r := newRecord()
-	cfgs, err := getConfigs(ctx, cOpts, opts, r)
+	cfgs, fieldPathElementsOverrodeByProfile, err := getConfigs(ctx, cOpts, opts, r)
```

Comment:
could this just be `fieldsOverrodeByProfile`? The name is a bit long/cumbersome right now and the meaning of `fieldPathElements` isn't entirely clear

Reply:
I have changed this to be s/fieldPathElementsOverrodeByProfile/fieldsOverrodeByProfile.  This was the initial name but I changed it when I realized I would need paths to fields (eg: /build/artifacts/0/image) vs the name of a field (image).  This was to let readers know that the keys are paths.  I am fine w/ this change though, it is done now